.gitignore

```diff
@@ -57,3 +57,5 @@ typings/
 # dotenv environment variables file
 .env
 
+#.DS_store files
+*.DS_Store
```

Comment:
For this, you don't need the * in front of `.DS_Store`. Also, the comment above (the line that starts with #) should be a more general description, like `# MacOS system files`.